playwright/accessibility/Typography.spec.mjs

```diff
@@ -3,6 +3,6 @@ import { test } from '@playwright/test';
 import expectAccessiblePage from './expectAccessiblePage.mjs';
 
 test('Typography Accessibility check', async ({ page }) => {
-  await page.goto('/web/typography');
```

Comment:
Shouldn't this fail? surprised it didn't complain

Reply:
Gooooood question